docker/quickstart/docker-compose.yml

```diff
@@ -191,18 +182,17 @@ services:
       - KAFKA_SCHEMAREGISTRY_URL=http://schema-registry:8081
       - ELASTICSEARCH_HOST=elasticsearch
       - ELASTICSEARCH_PORT=9200
+      - NEO4J_HOST=neo4j:7474
       - NEO4J_URI=bolt://neo4j
       - NEO4J_USERNAME=neo4j
       - NEO4J_PASSWORD=datahub
     depends_on:
-      - elasticsearch
-      - broker
+      - elasticsearch-setup
+      - kafka-setup
       - mysql
-      - schema-registry
       - neo4j
-    command: "sh -c 'dockerize -wait tcp://mysql:3306 -wait tcp://broker:29092 -wait http://elasticsearch:9200 \
-                                   -timeout 240s \
-                                   java -jar jetty-runner-9.4.20.v20190813.jar gms.war'"
+    command: "sh -c 'while ping -c1 kafka-setup &>/dev/null; do echo waiting for kafka-setup... && sleep 1; done; \
+                  echo kafka-setup done! && /start.sh'"
```

Comment:
GMS also sends MAE right? Wouldn't it need to wait for the topic creation as well?

Reply:
Actually, it doesn't because it's the producer. Even if topic is not created, when GMS starts to produce, topic will be automatically created because `auto.create.topics.enable` is true by default. http://kafka.apache.org/documentation.html#auto.create.topics.enable

However, topics are not created on Kafka, mxe consumers fails during initialization.

Reply:
Got it. Yeah auto topic creation doesn't work for Kafka stream apparently: https://docs.confluent.io/current/streams/developer-guide/manage-topics.html